packages/es-components/src/components/containers/popover/Popup.js

```diff
@@ -182,16 +181,28 @@ function Popup(props) {
         <Popper
           className={`${name}-popper`}
           placement={position}
-          modifiers={{
-            preventOverflow: { boundariesElement: document.body },
-            flip: {
-              enabled: !disableFlipping,
-              behavior: ['left', 'right', 'top', 'bottom', 'top']
+          modifiers={[
+            {
+              name: 'preventOverflow',
+              options: {
+                boundariesElement: document.body,
+                tether: keepTogether
+              }
             },
-            keepTogether: {
-              enabled: keepTogether
+            {
+              name: 'flip',
+              options: {
+                enabled: !disableFlipping,
+                behavior: ['left', 'right', 'top', 'bottom', 'top']
```

Comment:
@Darrken do you remember why we needed `top` listed here twice?

Reply:
I don't. Actually I'm not sure this is correct for popper 2.x -- `behavior` isn't listed in their docs as a modifier for 2.x any longer: https://popper.js.org/docs/v2/modifiers/flip/#options, and this might need to be changed to `fallbackPlacements`.

Reply:
I'm comfortable with these changes, but I want to make sure we're not worried about the popper update before we get merged. @Darrken, do you think we should make these updates you're suggesting. (Dmytro started a new job this week, so I don't think we can expect him to follow up on this.)

Reply:
We should pull this down and test it to make sure the popper behaves as expected with the suggested change. I think I'd feel comfortable with the changes if more than one of us has run through it locally. :)

Reply:
@Darrken @stevematney All our popover examples seem to be working fine. I also vote for someone else to pull down and check, just to make sure I didn't miss anything.

Reply:
I've seen these warnings once in a while. They don't seem to be harmful, but I'm not sure why they crop up.

Reply:
(I wonder if a styleguidist version bump would make them go away.)